readablepdf

```diff
@@ -354,13 +358,26 @@ while true; do
 	esac
 done
 
-# Use GNU Parallel to speed things up if it exists.
+
+echo "Estimating CPU-Threads"
+# Estimate CPU Cores (see pdfsandwich code for reference)
+CPU_THREADS=$(cat /proc/cpuinfo | grep processor | awk '{a++} END {print a}')
+OMP_THREAD_LIMIT=$CPU_THREADS
+MAX_JOBS=$(expr $CPU_THREADS / 4)
+# tesseract only uses 4 threads
+
+echo "Using up to $CPU_THREADS threads."
+
+# command -v foo >/dev/null 2>&1 || { echo >&2 "I require foo but it's not installed.  Aborting."; exit 1; }
+
 if command -v parallel >/dev/null; then
-	parallel tesseract "{}" "{.}" "${TESSERACT_PARAMS}" hocr ::: *.tif
-else
-	for i in ./*.tif; do tesseract "$i" "$(basename "$i")" "${TESSERACT_PARAMS}" hocr; done;
+	parallel -P ${MAX_JOBS} tesseract "{}" "{.}" "${TESSERACT_PARAMS}" hocr ::: *.tif
+ else
+	 for i in ./*.tif; do tesseract $i $(basename "$i") ${TESSERACT_PARAMS} hocr; done;
```

Comment:
It seems a couple of spaces snuck in, also a couple of newlines below that I'm not sure if intended. Otherwise lgtm and thanks! :+1: